systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaClientsResource.java

```diff
@@ -117,7 +72,7 @@ public static DoneableDeployment deployKafkaClients(boolean tlsListener, String
         return KubernetesResource.deployNewDeployment(kafkaClient);
     }
 
-    private static PodSpec createClientSpec(boolean tlsListener, String kafkaClientsName, boolean hostnameVerification, KafkaUser... kafkaUsers) {
+    private static PodSpec createClientSpec(boolean tlsListener, String kafkaClientsName, boolean hostnameVerification, KafkaUser...kafkaUsers) {
```

Comment:
why you removed a space here?

Reply:
Some "refactor" leftover and "miss click", thanks :)